test/integration/auth/mongodb_oidc.test.ts

```diff
@@ -0,0 +1,17 @@
+import { expect } from 'chai';
+
+describe('MONGODB-OIDC', function () {
```

Comment:
The full set of prose tests is coming in NODE-5035, but I wanted at least one test in this file to show the env got set up properly.